api/v1beta1/common.go

```diff
@@ -123,13 +118,24 @@ type NodeTemplate struct {
 	NetworkData *corev1.SecretReference `json:"networkData,omitempty"`
 }
 
-// NetworkConfigSection is a specification of the Network configuration details
-type NetworkConfigSection struct {
+// DeployStrategySection for fields controlling the deployment
+type DeployStrategySection struct {
```

Comment:
This section is also not relevant. Not rebased properlu.

Reply:
And this one. My fork wasn't in sync with upstream/main when I rebased. Fixed that now, so all of these issues should be resolved.